.changeset/sweet-lions-laugh.md

```diff
@@ -0,0 +1,5 @@
+---
+"@paprika/tag": patch
+---
+
+change tag styling
```

Comment:
Maybe update this message to `Tag Icon closing Icon (X icon) background color will be changed depending on the tag background`